recipes/libfabric/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "1.22.0" %}
+{% set build = 0 %}
+
+package:
+  name: libfabric
+  version: {{ version }}
+
+source:
+  url: https://github.com/ofiwg/libfabric/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 302bf1a3ef64bf51a41a1882f87277a7ab259c8790600e30cf78eee25dfa63dd
+
+build:
+  number: {{ build }}
+  skip: true  # [win or osx]
+
+  run_exports:
+    - {{ pin_subpackage('libfabric', max_pin='x.y') }}
+
+requirements:
+  build:
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - libnl  # [linux]
+    - libtool  # [unix]
+    - make  # [unix]
+    - numactl  # [linux]
+  host:
+  run:
+
+test:
+  commands:
+    - fi_info --list
+
+about:
+  home: http://libfabric.org/
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
```

the license file in the repo is [COPYING](https://github.com/ofiwg/libfabric/blob/v1.22.0/COPYING):

```suggestion
  license_file: COPYING
```